### ISSUE TITLE
AttributeError: 'CoxPHFitter' object has no attribute 'score'

### ISSUE THREAD

**Issue by seyuboglu** [CamDavidsonPilon/lifelines#1102]
Hi there!

My fit `CoxPHFitter` instances don't seem to have a `score` method. The documentation [here](https://lifelines.readthedocs.io/en/latest/fitters/regression/CoxPHFitter.html#lifelines.fitters.coxph_fitter.CoxPHFitter.score) suggests that they should. Perhaps I'm misunderstanding how to score a dataset with a `CoxPHFitter`?

Here is a simple example from the [documentation](https://lifelines.readthedocs.io/en/latest/fitters/regression/CoxPHFitter.html#lifelines.fitters.coxph_fitter.SemiParametricPHFitter.score) that raises the `AttributeError` I'm running into: 

```python
from lifelines import CoxPHFitter
from lifelines.datasets import load_rossi

rossi_train = load_rossi().loc[:400]
rossi_test = load_rossi().loc[400:]
cph = CoxPHFitter().fit(rossi_train, 'week', 'arrest')

cph.score(rossi_train)
cph.score(rossi_test)

>>>
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-201-88ced0ed8d15> in <module>
      4 rossi = load_rossi()
      5 cph = CoxPHFitter().fit(rossi, 'week', 'arrest')
----> 6 cph.score(rossi)

AttributeError: 'CoxPHFitter' object has no attribute 'score'
```

Thank you for the help!

**Comment by seyuboglu** [comment]
solved: Upgraded lifelines from `0.23.0` to `0.25.1` and restarted my Jupyter kernel.

**Comment by CamDavidsonPilon** [comment]
Happy you figured it out, @seyuboglu!
